Modules/ITS/src/ITSTrackSimTask.cxx

```diff
@@ -87,19 +87,18 @@ ITSTrackSimTask::~ITSTrackSimTask()
 
   delete hTrackImpactTransvFake;
   delete hTrackImpactTransvValid;
+
+  delete hPrimaryReco_pt;
+  delete hPrimaryGen_pt;
 }
 
 void ITSTrackSimTask::initialize(o2::framework::InitContext& /*ctx*/)
 {
   ILOG(Info, Support) << "initialize ITSTrackSimTask" << ENDM;
 
-  mRunNumberPath = mCustomParameters["runNumberPath"];
-  mMCKinePath = mCustomParameters["MCKinePath"];
-  mO2GrpPath = mCustomParameters["o2GrpPath"];
-
   createAllHistos();
   publishHistos();
-  o2::base::Propagator::initFieldFromGRP(mO2GrpPath.c_str());
+  o2::base::Propagator::initFieldFromGRP("o2sim_grp.root");
```

Comment:
Isn't it better to have it in the json as before in case the file is not in the current directory? Or maybe this initFieldFromGRP does the readout of the file automatically?